### PR TITLE
{tools}[GCCcore/11.3.0] cppy v1.2.1 w/ Python 3.10.4 WIB

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/cppy/cppy-1.2.1-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# we are using PyPi for this:
+# source_urls = ['https://github.com/nucleic/cppy/archive/refs/tags/']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['83b43bf17b1085ac15c5debdb42154f138b928234b21447358981f69d0d6fe1b']
+
+download_dep_fail = True
+sanity_pip_check = True
+use_pip = True
+
+moduleclass = 'tools'
```
Comment (jfgrimm) [inline easybuild/easyconfigs/c/cppy/cppy-1.2.1-GCCcore-11.2.0.eb:30]:
```suggestion
easyblock = 'PythonBundle'

name = 'cppy'
version = '1.2.1'

homepage = "https://github.com/nucleic/cppy"
description = """A small C++ header library which makes it easier to write
Python extension modules. The primary feature is a PyObject smart pointer
which automatically handles reference counting and provides convenience
methods for performing common object operations."""

toolchain = {'name': 'GCCcore', 'version': '11.2.0'}

dependencies = [('Python', '3.9.6')]

use_pip = True

exts_list = [
    ('setuptools', '62.6.0', {
        'checksums': ['990a4f7861b31532871ab72331e755b5f14efbe52d336ea7f6118144dd478741'],
    }),
    (name, version, {
        'checksums': ['83b43bf17b1085ac15c5debdb42154f138b928234b21447358981f69d0d6fe1b'],
    }),
]

sanity_pip_check = True

moduleclass = 'tools'
```

**Comment by sassy-crick** [comment]
Thanks for finding this! As discussed, I will update this to `foss-2022a` so we don't have too many `cppy` in the same toolchain.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**